server/repository/src/migrations/v1_06_00/master_list_line_add_item_link_id.rs

```diff
@@ -0,0 +1,41 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    if cfg!(feature = "postgres") {
```

Comment:
I think preference should be for build time checks to keep bundles negligibly (lol) smaller: `#[cfg(feature = "postgres")]`